test/controllers/repos_controller_test.rb

```diff
@@ -1,10 +1,38 @@
 require 'test_helper'
 
 class ReposControllerTest < ActionController::TestCase
+  setup do
+    @repo = repos(:word_guess)
+  end
+
+  def create_params
+    {
+      repo: {
+        repo_url: "test"
+      }
+    }
+  end
+
+  def create_params_invalid
+    create_params.deep_merge(repo: { repo_url: nil })
```

Comment:
Would it be more efficient to create a brand new hash since none of the properties are shared?

Reply:
Probably. I think I wrote this with the intent that if we add more params later then we wouldn't need to update it in two places. But since that's not really that likely, I'll simplify this.